### PR TITLE
Update resnet.py

### PR DESCRIPTION
**Pull request by mahdiabavisani**
Test with mnist test set. Previously it was testing on the training set.

### CODE REVIEW

**Comment by mahdiabavisani** [comment]
I signed it!

**Review by terrytangyuan — APPROVED** [review]

**Comment by gunan** [comment]
Jenkins, test this please.

**Review by vrv — APPROVED** [review]
